app/services/encryption/kms_logger.rb

```diff
@@ -5,6 +5,7 @@ class KmsLogger
     def self.log(action, key_id:, context: nil, log_context: nil)
       output = {
         kms: {
+          timestamp: Time.zone.now,
```

Comment:
```suggestion
    def self.log(action:, timestamp:, key_id:, context: nil, log_context: nil)
      output = {
        kms: {
          timestamp: timestamp,
```

I wonder if it might be preferable to pass the time as an argument? Would have some implications around the tests, but I suspect they will require freezing time.